packages/office-ui-fabric-react/src/components/Dropdown/examples/Dropdown.ControlledMulti.Example.tsx

```diff
@@ -1,56 +1,45 @@
+const DropdownControlledMultiExampleOptions = [
+  { key: 'fruitsHeader', text: 'Fruits', itemType: DropdownMenuItemType.Header },
+  { key: 'apple', text: 'Apple' },
+  { key: 'banana', text: 'Banana' },
+  { key: 'orange', text: 'Orange', disabled: true },
+  { key: 'grape', text: 'Grape' },
+  { key: 'divider_1', text: '-', itemType: DropdownMenuItemType.Divider },
+  { key: 'vegetablesHeader', text: 'Vegetables', itemType: DropdownMenuItemType.Header },
+  { key: 'broccoli', text: 'Broccoli' },
+  { key: 'carrot', text: 'Carrot' },
+  { key: 'lettuce', text: 'Lettuce' },
+];
 
-export class DropdownControlledMultiExample extends React.Component<{}, IDropdownControlledMultiExampleState> {
-  public state: IDropdownControlledMultiExampleState = {
-    selectedItems: [],
-  };
-
-  public render() {
-    const { selectedItems } = this.state;
+export const DropdownControlledMultiExample: React.FC = () => {
+  const [selectedItems, setSelectedItems] = React.useState();
 
-    return (
-      <Dropdown
-        placeholder="Select options"
-        label="Multi-select controlled example"
-        selectedKeys={selectedItems}
-        onChange={this._onChange}
-        multiSelect
-        options={[
-          { key: 'fruitsHeader', text: 'Fruits', itemType: DropdownMenuItemType.Header },
-          { key: 'apple', text: 'Apple' },
-          { key: 'banana', text: 'Banana' },
-          { key: 'orange', text: 'Orange', disabled: true },
-          { key: 'grape', text: 'Grape' },
-          { key: 'divider_1', text: '-', itemType: DropdownMenuItemType.Divider },
-          { key: 'vegetablesHeader', text: 'Vegetables', itemType: DropdownMenuItemType.Header },
-          { key: 'broccoli', text: 'Broccoli' },
-          { key: 'carrot', text: 'Carrot' },
-          { key: 'lettuce', text: 'Lettuce' },
-        ]}
-        styles={{ dropdown: { width: 300 } }}
-      />
-    );
-  }
-
-  private _onChange = (event: React.FormEvent<HTMLDivElement>, item: IDropdownOption): void => {
-    const newSelectedItems = [...this.state.selectedItems];
+  const onChange = (event: React.FormEvent<HTMLDivElement>, item: IDropdownOption): void => {
+    setSelectedItems(item);
+    const newSelectedItems = selectedItems;
     if (item.selected) {
-      // add the option if it's checked
       newSelectedItems.push(item.key as string);
     } else {
-      // remove the option if it's unchecked
       const currIndex = newSelectedItems.indexOf(item.key as string);
       if (currIndex > -1) {
         newSelectedItems.splice(currIndex, 1);
       }
     }
-    this.setState({
-      selectedItems: newSelectedItems,
-    });
+    setSelectedItems(newSelectedItems);
   };
-}
+
+  return (
+    <Dropdown
+      placeholder="Select options"
+      label="Multi-select controlled example"
+      selectedKeys={selectedItems}
+      onChange={onChange}
+      multiSelect
+      options={DropdownControlledMultiExampleOptions}
+      styles={{ dropdown: { width: 300 } }}
```

Comment:
there is going to be some conflicts with this pr: https://github.com/microsoft/fluentui/pull/12631 :)

Reply:
Thanks for reviewing!  I updated the examples in this PR to avoid mutating styles props.